frontend/src/metabase/components/ExplicitSize/ExplicitSize.tsx

```diff
@@ -75,8 +74,8 @@ function ExplicitSize<T extends BaseInnerProps>({
         };
 
         this._printMediaQuery = window.matchMedia && window.matchMedia("print");
-        if (this.context) {
-          this._refreshMode = this.context as RefreshMode;
+        if (this.context === 0) {
```

Comment:
This is more or less the same behavior, but I change the provider to work with another location that uses wait time for UI, so we don't have to have too many contexts.